designs/administrator/4.4/quay-imagemanifestvuln-list/index.md

```diff
@@ -29,11 +29,4 @@ Enhancements have been made to the **Image Vulnerabilities** status on the clust
 
 ![ImageManifestVulnerabilities List Filtered Pod](img/5filteredimvlist.png)
 - The user is taken to the **Image Vulnerabilities** list, with a filter applied to only show the vulnerability(s) that relate to that pod. The list may be empty if no vulnerability exists for that pod's image(s).
```

Comment:
What do we think about adding vulnerability information to the pod list view directly? Maybe in the form of an icon that doesn't occupy a lot of space and only appears if a vulnerability is actually found so we don't clutter the UI?

Reply:
Perhaps we extend the Status column and use a popover? Something like "Running - Vulnerable" with an appropriate icon?

Reply:
This seems like a good idea to me. @dmesser maybe something we could discuss at our meeting on Monday as something to try for 4.6? I added it to my [list of potential improvements](https://docs.google.com/document/d/1yrzPFnH6KeCqfwNt3g_B8UsjmYr0MTCn4jQydIahmU0/edit?usp=sharing).